src/main/java/com/example/demo/WebController.java

```diff
@@ -51,7 +51,7 @@ public Admin loginAdmin(@RequestBody String logInfo) throws Exception {
         return adminService.login(input.getEmail(),input.getPassword());
     }
 
-    @PostMapping(value = "/add")
+    @PostMapping(value = "/applications/add")
```

Comment:
This should be just `@PostMapping(value="/applications")` - in CRUD, POST is "Create/Add".